### DISCUSSION TITLE
Edge z-index?

### DISCUSSION THREAD

**Discussion by baptisteArno** [xyflow/xyflow#1709] [Q&A] [answered]
My handle is inside a Node, is it possible to draw the Edge on top of it?

![CleanShot 2021-11-23 at 10 43 53@2x](https://user-images.githubusercontent.com/16015833/143002147-841ad3e9-3c66-4490-a939-cd181f0ce82b.png)

**Comment by moklick** [comment, marked as answer]
Hi @baptisteArno

You can increase the z-index of the edge renderer (the node-renderer has a z-index of 3): 

```css
.react-flow .react-flow__edges {
    z-index: 4;
}
```

or you could wait for the next major release where we introduce a z-index per edge ;)

**Reply by ks2211** [reply]
Hey @moklick has the z-index per edge feature been added? If so which version(I can't seem to get a z-index based on the edge class to work)

**Reply by moklick** [reply]
It's still in beta but you can find the installation instructions and changes for v10 in this PR #1555
